docs/details/shards/GFX/UIPass.md

```diff
@@ -0,0 +1,3 @@
+This defines a render step for rendering [UI](../../General/UI)
+
+Check the examples on some of the UI shards for how to use this step, for example [UI.Label](../../UI/Label)
```

Comment:
Might be better to directly insert the examples here instead of linking it elsewhere for usage examples.